### PR TITLE
IN -  Control anular internación

### PR DESCRIPTION
**Pull request by aldoEMatamala**
### Requerimiento
 
https://proyectos.andes.gob.ar/browse/IN-532

https://github.com/andes/app/pull/2853
### Funcionalidad desarrollada 

1. Modificada leyenda al "deshacer internación" 

### Requiere actualizaciones en la base de datos

- [ ] Si
- [x] No

### CODE REVIEW

**Comment by silviroa** [comment]
USUARIO: amatamala  
BUILD NUMBER: [9266](http://condorpiedra.andes.gob.ar:85/9266/mochawesome.html)
CYPRESS RUN: [8412](https://dashboard.cypress.io/projects/xr7gft/runs/8412/specs)
TEST START: 2023-07-25T18:18:01.308Z
TOTAL: 403
SUCCESS: 401
FAIL: 0
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: jramirez  
BUILD NUMBER: [9436](http://condorpiedra.andes.gob.ar:85/9436/mochawesome.html)
CYPRESS RUN: [8573](https://dashboard.cypress.io/projects/xr7gft/runs/8573/specs)
TEST START: 2023-08-16T13:56:29.425Z
TOTAL: 403
SUCCESS: 400
FAIL: 1
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: jramirez  
BUILD NUMBER: [9437](http://condorpiedra.andes.gob.ar:85/9437/mochawesome.html)
CYPRESS RUN: [8574](https://dashboard.cypress.io/projects/xr7gft/runs/8574/specs)
TEST START: 2023-08-16T14:09:17.101Z
TOTAL: 361
SUCCESS: 350
FAIL: 9
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: jramirez  
BUILD NUMBER: [9447](http://condorpiedra.andes.gob.ar:85/9447/mochawesome.html)
CYPRESS RUN: [8578](https://dashboard.cypress.io/projects/xr7gft/runs/8578/specs)
TEST START: 2023-08-16T16:14:01.671Z
TOTAL: 360
SUCCESS: 348
FAIL: 10
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: jramirez  
BUILD NUMBER: [9448](http://condorpiedra.andes.gob.ar:85/9448/mochawesome.html)
CYPRESS RUN: [8579](https://dashboard.cypress.io/projects/xr7gft/runs/8579/specs)
TEST START: 2023-08-16T16:29:46.832Z
TOTAL: 403
SUCCESS: 399
FAIL: 2
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: jramirez  
BUILD NUMBER: [9453](http://condorpiedra.andes.gob.ar:85/9453/mochawesome.html)
CYPRESS RUN: [8583](https://dashboard.cypress.io/projects/xr7gft/runs/8583/specs)
TEST START: 2023-08-16T18:10:34.258Z
TOTAL: 353
SUCCESS: 349
FAIL: 2
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: jramirez  
BUILD NUMBER: [9458](http://condorpiedra.andes.gob.ar:85/9458/mochawesome.html)
CYPRESS RUN: [8588](https://dashboard.cypress.io/projects/xr7gft/runs/8588/specs)
TEST START: 2023-08-17T11:31:01.380Z
TOTAL: 403
SUCCESS: 400
FAIL: 1
SKIPPED: 2

File: cypress/e2e/apps/rup/internacion/mapa-camas/detalle-cama.spec.ts [hunk 1]
```diff
@@ -162,6 +162,8 @@ describe('Mapa Camas - Detalle de Cama', () => {
 
         cy.swal('confirm');
     });
+
+
```
Comment (MarianoCampetella) [inline cypress/e2e/apps/rup/internacion/mapa-camas/detalle-cama.spec.ts:166]: Quitar los nuevos espacios y amendearlo.

**Review by MarianoCampetella — CHANGES_REQUESTED** [review]

File: cypress/e2e/apps/rup/internacion/mapa-camas/detalle-cama.spec.ts (hunk 1 above)
Comment (aldoEMatamala) [reply cypress/e2e/apps/rup/internacion/mapa-camas/detalle-cama.spec.ts]: amendeado gracias!

**Comment by silviroa** [comment]
USUARIO: amatamala  
BUILD NUMBER: [9487](http://condorpiedra.andes.gob.ar:85/9487/mochawesome.html)
CYPRESS RUN: [8614](https://dashboard.cypress.io/projects/xr7gft/runs/8614/specs)
TEST START: 2023-08-22T12:46:44.882Z
TOTAL: 403
SUCCESS: 400
FAIL: 1
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: amatamala  
BUILD NUMBER: [9489](http://condorpiedra.andes.gob.ar:85/9489/mochawesome.html)
CYPRESS RUN: [8616](https://dashboard.cypress.io/projects/xr7gft/runs/8616/specs)
TEST START: 2023-08-22T13:16:35.935Z
TOTAL: 403
SUCCESS: 401
FAIL: 0
SKIPPED: 2

**Review by JuanIRamirez — APPROVED** [review]
ok

**Comment by silviroa** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [9493](http://condorpiedra.andes.gob.ar:85/9493/mochawesome.html)
CYPRESS RUN: [8620](https://dashboard.cypress.io/projects/xr7gft/runs/8620/specs)
TEST START: 2023-08-23T11:46:09.848Z
TOTAL: 403
SUCCESS: 401
FAIL: 0
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: amatamala  
BUILD NUMBER: [9494](http://condorpiedra.andes.gob.ar:85/9494/mochawesome.html)
CYPRESS RUN: [8621](https://dashboard.cypress.io/projects/xr7gft/runs/8621/specs)
TEST START: 2023-08-23T11:56:45.322Z
TOTAL: 360
SUCCESS: 349
FAIL: 9
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: amatamala  
BUILD NUMBER: [9495](http://condorpiedra.andes.gob.ar:85/9495/mochawesome.html)
CYPRESS RUN: [8622](https://dashboard.cypress.io/projects/xr7gft/runs/8622/specs)
TEST START: 2023-08-23T12:16:55.944Z
TOTAL: 368
SUCCESS: 359
FAIL: 7
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: amatamala  
BUILD NUMBER: [9498](http://condorpiedra.andes.gob.ar:85/9498/mochawesome.html)
CYPRESS RUN: [8625](https://dashboard.cypress.io/projects/xr7gft/runs/8625/specs)
TEST START: 2023-08-24T11:14:11.315Z
TOTAL: 403
SUCCESS: 401
FAIL: 0
SKIPPED: 2

**Review by MarianoCampetella — APPROVED** [review]

**Pull request merged**